### PR TITLE
Fix: remove the check on the iam addon.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Iam addon is removed via this PR from klusterlet-addon-controller repo: https://github.com/stolostron/klusterlet-addon-controller/pull/227

### CODE REVIEW

File: pkg/utils/cluster.go [hunk 1]
```diff
@@ -18,22 +28,12 @@ import (
...
-	libgocrdv1 "github.com/stolostron/library-go/pkg/apis/meta/v1/crd"
-	libgodeploymentv1 "github.com/stolostron/library-go/pkg/apis/meta/v1/deployment"
-	libgounstructuredv1 "github.com/stolostron/library-go/pkg/apis/meta/v1/unstructured"
 )
 
 // list of manifestwork name for addon crs
 var managedClusteraddOns = []string{
 	"application-manager",
 	"cert-policy-controller",
-	"iam-policy-controller",
```
Comment (xuezhaojun) [inline pkg/utils/cluster.go:36]: @zhiweiyin318 Only change, other lines are because fmt.

**Comment by dhaiducek** [comment]
@zhiweiyin318 @xuezhaojun It looks like fast-forwarding for this repo wasn't updated and this got fast-forwarded to release-2.8, which isn't correct:
```
      DESTINATION_BRANCH: release-2.8
```
https://github.com/openshift/release/blob/b1b699bc90d4d1d0120fdc058e726e97b4ea3f66/ci-operator/config/stolostron/cluster-lifecycle-e2e/stolostron-cluster-lifecycle-e2e-main.yaml#L70

I think we'll have to:
1. revert the change in release-2.8 since iam-policy-controller still exists there
2. update fast-forwarding to release-2.11 (or delete that config if this repo doesn't receive many updates and should use manual cherry-picks instead)
4. cherry-pick this PR to release-2.11 (or fast-forward release-2.11, depending on the choice in step 2)

**Comment by zhiweiyin318** [comment]
@dhaiducek  thanks. 
have fixed the ff issue and reverted the PR on 0.8.

**Pull request merged**